lib/interface_quda.cpp

```diff
@@ -8876,119 +8872,46 @@ void calcMG_loop_wOneD_TSM_wExact(void **gaugeToPlaquette,
     // in -> b, out -> x, for parity singlets
```

Comment:
maybe we should set "out" to zero here, to make sure that initial guess is zero for the inversion, i.e.:
zeroCuda(*out);